README.md

```diff
@@ -0,0 +1,252 @@
+# ACCESS-ESM with PAYU
+
+## Understanding PAYU
+
+PAYU was designed to help users of the NCI system run climate models.
+It was initially created for MOM, but has been adapted for other models,
+including coupled ones.
+
+The aim of PAYU is to make it easy and intuitive to configure and run the models.
+
+PAYU knows certain models and how to run them. Adding more models needs additions to the PAYU sources.
+This will not be part of this document.
+
+### Terms
+
+To understand PAYU, it helps to distinguish certain terms:
+
+-   The **Laboratory** is a directory where all parts of the model are kept.
```

Comment:
Because terms like Laboratory are used as keywords, probably best not to capitalise.